tests/__Init__.py

```diff
@@ -0,0 +1,6 @@
+import os
```

Comment:
I guess it won't be a problem but this file should be renamed to lowercase __init__.py

Reply:
I will fix the file name of this 👌